File: Assignments/Assignment_5/program4.c
// Check positive, negative or zero

#include<stdio.h>

////////////////////////////////////////////////////////////////////
//
//  Function Name : CheckNumberType
//  Description :   Check positive, Negative, or zero
//  Input :         Integer
//  Output :        Integer
//  Author :        Omkar Sachin Narlawar
//  Date :          26/10/2025
//
////////////////////////////////////////////////////////////////////

void CheckNumberType(int num)
{
    if(num < 0)
    {
        printf("Number is Negative");
    }
    else if(num > 0)
    {
        printf("Number is Positive");
    }
    else
    {
        printf("Number is Zero");
    }
}

int main()
{
    int Number = 0;

    printf("Enter number\n");
    scanf("%d",&Number);

    CheckNumberType(Number);

    return 0;
}

File: Assignments/Assignment_28/Java_Programs/program4.java
/* Accept number of rows and columns from user and display below pattern.
 Input : iRow = 4     iCol = 5
 OUTPUT :
            4   4   4   4   4
            3   3   3   3   3
            2   2   2   2   2
            1   1   1   1   1
*/

////////////////////////////////////////////////////////////////////////
//
//  Required Import File
//
////////////////////////////////////////////////////////////////////////

import java.util.*;

////////////////////////////////////////////////////////////////////////
//
//  Function Name   :   Display
//  Description     :   used to print patterns
//  Input           :   Integer
//  Output          :   Pattern
//  Author          :   Omkar Sachin Narlawar
//  Date            :   22/11/2025
//
////////////////////////////////////////////////////////////////////////

class Pattern
{
    public void Display(int iRow,int iCol)
    {
        int i = 0, j = 0;    

        for(i = iRow; i >= 1; i--)
        {
            for(j = 1; j <= iCol; j++)
            {
                System.out.print(i+"\t");
            }
            System.out.println();
        }
    }
}// End of Class

////////////////////////////////////////////////////////////////////////
//
//  Entry point function : Main
//
////////////////////////////////////////////////////////////////////////

class program4
{
    public static void main(String A[])
    {
        int iValue1 = 0, iValue2 = 0;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number of Rows : ");
        iValue1 = sobj.nextInt();

        System.out.println("Enter number of Columns : ");
        iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();
        pobj.Display(iValue1,iValue2);
    }
}// End of Main